holmes/plugins/toolsets/coralogix/toolset_coralogix_logs.py

```
@@ -26,80 +24,69 @@
     build_coralogix_link_to_logs,
     stringify_flattened_logs,
 )
-from holmes.plugins.toolsets.utils import standard_start_datetime_tool_param_description
+from holmes.plugins.toolsets.logging_api import (
+    BasePodLoggingToolset,
+    FetchPodLogsParams,
+    PodLoggingTool,
+)
+from holmes.plugins.toolsets.utils import (
+    TOOLSET_CONFIG_MISSING_ERROR,
```

Ruff (0.11.9)

Redefinition of unused `TOOLSET_CONFIG_MISSING_ERROR` from line 14

(F811)